packages/migrations/src/actions/2025.03.20T00-00-00.dangerous_breaking.ts

```diff
@@ -0,0 +1,12 @@
+import type { MigrationExecutor } from '../pg-migrator';
+
+export default {
+  name: '2025.03.20T00-00-00.dangerous_breaking.ts',
+  noTransaction: true,
+  run: ({ sql }) => sql`
+    ALTER TABLE
+      targets
+    ADD COLUMN
+      fail_dangerous_checks BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
NIT: `fail_diff_on_dangerous_change` would be a more descriptive naming (would also include renaming the code variables)